scapy/config.py

```diff
@@ -613,7 +613,7 @@ class Conf(ConfClass):
     netcache = NetCache()
     geoip_city = None
     load_layers = ['bluetooth', 'bluetooth4LE', 'dhcp', 'dhcp6', 'dns',
-                   'dot11', 'dot15d4', 'eap', 'gprs', 'hsrp', 'inet',
+                   'dot11', 'dot15d4', 'eap', 'geneve', 'gprs', 'hsrp', 'inet',
```

Comment:
`geneve` is a contrib module, and therefore shouldn't be loaded by default

Reply:
How should it be loaded?

Reply:
Either `load_contrib("geneve")` or `from scapy.contrib.geneve import *`